jobs/doppler/templates/doppler.json.erb

```diff
@@ -55,6 +55,8 @@
         if_p("doppler.blacklisted_syslog_ranges") do |prop|
             a[:BlackListIPs] = prop
         end
+        a[:BatchSize] = p('doppler.batch.size')
+        a[:Interval] = p('doppler.batch.interval')
```

Comment:
Shouldn't this be `a[:BatchInterval]` to match the field name in the config?

Reply:
Yep. Good catch. Same below.